.github/workflows/release-erroranalysis.yml

```diff
@@ -21,17 +21,26 @@ jobs:
 
       # build wheel
       - uses: actions/checkout@v2
+
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
           python-version: 3.7
+
+      - name: update and upgrade pip, setuptools, wheel, and twine
```

Comment:
Just curios, why did not fail during previous failures?

Reply:
during previous releases?  rai-core-flask was modified since last release.  erroranalysis has always been released from local machine and not github action.